portfolio/src/main/webapp/style.css

```diff
@@ -33,29 +49,34 @@ p {
     position: fixed;
     width: 100%;
     z-index: 100;
-    line-height: 80px;
+    transition: 0.5s;
```

Comment:
What transition is this? Something to do with clicking the links?

Reply:
My bad, this shouldn't be there. This is a transition for my shrink nav bar JS function, but I wanted to wait until these current changes had been reviewed first before including the function in my next commit.